src/main/java/mekanism/common/tile/TileEntityGasTank.java

```diff
@@ -421,4 +423,26 @@ public TileComponentSecurity getSecurity()
 	{
 		return securityComponent;
 	}
+
+	@Override
+	public String[] getMethods()
+	{
+		return methods;
+	}
+
+	@Override
+	public Object[] invoke(int method, Object[] arguments) throws Exception
+	{
+		switch(method)
+		{
+			case 0:
+				return new Object[] {gasTank.getMaxGas()};
+			case 1:
+				return new Object[] {gasTank.getStored()};
+			case 2:
+				return new Object[] {gasTank.getGas()};
+			default:
+				throw new NoSuchMethodException();
```

Comment:
Consistency in code is important, so people don't wonder why it's one way one place, and different somewhere else. If the devs agree with you, I expect they'd be happy to accept a refactoring change after this is merged (as an object return) to change them all to exceptions.

Just my non-dev opinion.